src/Core/SettingsChangesHistory.cpp

```diff
@@ -45,7 +45,7 @@ const VersionToSettingsChangesMap & getSettingsChangesHistory()
             {"input_format_protobuf_oneof_presence", false, false, "New setting"},
             {"iceberg_delete_data_on_drop", false, false, "New setting"},
             {"use_skip_indexes_on_data_read", false, false, "New setting"},
-            {"s3_slow_all_threads_after_retryable_error", true, true, "Added an alias for setting `backup_slow_all_threads_after_retryable_s3_error`"},
```

Comment:
Looks like you missed updating here, https://github.com/ClickHouse/ClickHouse/blob/master/src/Core/Settings.cpp#L472

Reply:
Thanks, fixed.